engine/src/main/java/org/hibernate/validator/internal/engine/valueextraction/ValueExtractorManager.java

```diff
@@ -155,19 +160,17 @@ else if ( maximallySpecificContainerElementCompliantValueExtractors.size() == 1
 	/**
 	 * Returns the maximally specific type-compliant and container-element-compliant value extractor
 	 * from a set of preselected value extractors or {@code null} if none was found.
-	 * <p>
```

Comment:
It's not a first time you remove a `<p>`, what's the rationale for this?

Reply:
It's the IDE reformatting things in such way. I finally found that setting and checked it so it shouldn't do it anymore. I'll send a PR for the codestyle repo with the update for that.